src/lmnt/api.py

```diff
@@ -77,13 +81,142 @@ async def close(self):
       await self._session.close()
     self._session = None
 
-  async def list_voices(self):
+
+  async def list_voices(self, starred: bool = False, owner: str = 'all'):
+    """
+    Returns a list of voices available to you.
+
+    Optional parameters:
+    - `starred`: Show starred voices only. Defaults to `false`.
+    - `owner`: Comma-separated list of voice owners to specify which voices to return. Choose from `lmnt`, `me`, or `all`. Defaults to `all`.
+    """
+    if owner not in ['all', 'me']:
+      raise ValueError(f'Invalid owner: {owner}')
     self._lazy_init()
-    url = f'{self._base_url}{_VOICES_ENDPOINT}'
+    url = f'{self._base_url}{_LIST_VOICES_ENDPOINT}?starred={starred}&owner={owner}'
+    
+    async with self._session.get(url, headers=self._build_headers()) as resp:
+      await self._handle_response_errors(resp)
+      return await resp.json()
+    
+
+  async def voice_info(self, voice_id):
+    """
+    Returns details of a specific voice.
+
+    Required parameters:
+    - `voice_id`: The id of the voice to update. If you don't know the id, you can get it from `list_voices()`.
+    """
+    self._lazy_init()
+    url = f'{self._base_url}{_VOICE_ENDPOINT}'.format(id=voice_id)
 
     async with self._session.get(url, headers=self._build_headers()) as resp:
       await self._handle_response_errors(resp)
-      return (await resp.json())['voices']
+      return await resp.json()
+
+
+  async def create_voice(self, name: str, enhance: bool, filenames: list[str], type: str = 'instant', gender: str = None, description: str = None):
+    """
+    Create a new voice from a set of audio files. Returns the voice metadata object.
+
+    Parameters:
+    - `name`: The name of the voice.
+    - `enhance`: Whether to enhance the audio files.
```

Comment:
Maybe elaborate a bit for clarity on this one, like, "for unclean audio with background noise, applies processing to attempt to improve quality. Not on by default as it can also degrade quality in some circumstances."

Also maybe explicitly state it's a boolean.

What made us make this required? Seems like we could make it optional and default to false (and then not throw error if it's None). Assume consistency with other SDKs/REST?

Reply:
mostly for consistency with REST